src/symex/symex_parse_options.cpp

```diff
@@ -364,7 +364,6 @@ bool symex_parse_optionst::process_goto_program(const optionst &options)
     remove_vector(goto_model);
     // Java virtual functions -> explicit dispatch tables:
     remove_virtual_functions(goto_model);
-    // Java instanceof -> clsid comparison:
```

Comment:
@tautschnig The exception processing is now called in symex, goto-analyzer and goto-instrument (I've actually introduced the call in symex_parse_options.cpp after your initial comments but was slightly misplaced).